<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="D-SHC-00436690" name="com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest" tests="6" failures="6" timestamp="23 Jan 2013 08:07:01 GMT" time="126.374" errors="0">
  <testcase name="checkTxnTest" time="21.273" classname="com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest">
    <failure type="java.lang.RuntimeException" message="An error occured while setting up the test environment. 
Root cause: ">
      <![CDATA[java.lang.RuntimeException: An error occured while setting up the test environment. 
Root cause: 
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:132)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:606)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:674)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.2.1', revision: '16551', time: '2012-04-11 21:42:35'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0'
Driver info: driver.version: ScreenShotRemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:454)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:140)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:95)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:103)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.<init>(ScreenShotRemoteWebDriver.java:62)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.createInstance(ScreenShotRemoteWebDriver.java:400)
	at com.paypal.test.bluefin.platform.grid.Grid.startSession(Grid.java:71)
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:89)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://qa-autocon26.sc4.paypal.com:4444 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:303)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:282)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:432)
	... 18 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:75)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:337)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:198)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:180)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:157)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
]]>
    </failure>
  </testcase> <!-- checkTxnTest -->
  <testcase name="keyTxnTest" time="21.022" classname="com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest">
    <failure type="java.lang.RuntimeException" message="An error occured while setting up the test environment. 
Root cause: ">
      <![CDATA[java.lang.RuntimeException: An error occured while setting up the test environment. 
Root cause: 
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:132)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:606)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:674)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.2.1', revision: '16551', time: '2012-04-11 21:42:35'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0'
Driver info: driver.version: ScreenShotRemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:454)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:140)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:95)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:103)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.<init>(ScreenShotRemoteWebDriver.java:62)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.createInstance(ScreenShotRemoteWebDriver.java:400)
	at com.paypal.test.bluefin.platform.grid.Grid.startSession(Grid.java:71)
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:89)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://qa-autocon26.sc4.paypal.com:4444 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:303)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:282)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:432)
	... 18 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:75)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:337)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:198)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:180)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:157)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
]]>
    </failure>
  </testcase> <!-- keyTxnTest -->
  <testcase name="track2TxnTest" time="21.019" classname="com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest">
    <failure type="java.lang.RuntimeException" message="An error occured while setting up the test environment. 
Root cause: ">
      <![CDATA[java.lang.RuntimeException: An error occured while setting up the test environment. 
Root cause: 
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:132)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:606)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:674)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.2.1', revision: '16551', time: '2012-04-11 21:42:35'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0'
Driver info: driver.version: ScreenShotRemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:454)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:140)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:95)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:103)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.<init>(ScreenShotRemoteWebDriver.java:62)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.createInstance(ScreenShotRemoteWebDriver.java:400)
	at com.paypal.test.bluefin.platform.grid.Grid.startSession(Grid.java:71)
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:89)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://qa-autocon26.sc4.paypal.com:4444 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:303)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:282)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:432)
	... 18 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:75)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:337)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:198)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:180)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:157)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
]]>
    </failure>
  </testcase> <!-- track2TxnTest -->
  <testcase name="track12TxnTest" time="21.026" classname="com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest">
    <failure type="java.lang.RuntimeException" message="An error occured while setting up the test environment. 
Root cause: ">
      <![CDATA[java.lang.RuntimeException: An error occured while setting up the test environment. 
Root cause: 
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:132)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:606)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:674)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.2.1', revision: '16551', time: '2012-04-11 21:42:35'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0'
Driver info: driver.version: ScreenShotRemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:454)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:140)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:95)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:103)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.<init>(ScreenShotRemoteWebDriver.java:62)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.createInstance(ScreenShotRemoteWebDriver.java:400)
	at com.paypal.test.bluefin.platform.grid.Grid.startSession(Grid.java:71)
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:89)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://qa-autocon26.sc4.paypal.com:4444 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:303)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:282)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:432)
	... 18 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:75)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:337)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:198)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:180)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:157)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
]]>
    </failure>
  </testcase> <!-- track12TxnTest -->
  <testcase name="track1TxnTest" time="21.018" classname="com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest">
    <failure type="java.lang.RuntimeException" message="An error occured while setting up the test environment. 
Root cause: ">
      <![CDATA[java.lang.RuntimeException: An error occured while setting up the test environment. 
Root cause: 
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:132)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:606)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:674)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.2.1', revision: '16551', time: '2012-04-11 21:42:35'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0'
Driver info: driver.version: ScreenShotRemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:454)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:140)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:95)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:103)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.<init>(ScreenShotRemoteWebDriver.java:62)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.createInstance(ScreenShotRemoteWebDriver.java:400)
	at com.paypal.test.bluefin.platform.grid.Grid.startSession(Grid.java:71)
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:89)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://qa-autocon26.sc4.paypal.com:4444 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:303)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:282)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:432)
	... 18 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:75)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:337)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:198)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:180)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:157)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
]]>
    </failure>
  </testcase> <!-- track1TxnTest -->
  <testcase name="chipTxnTest" time="21.016" classname="com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest">
    <failure type="java.lang.RuntimeException" message="An error occured while setting up the test environment. 
Root cause: ">
      <![CDATA[java.lang.RuntimeException: An error occured while setting up the test environment. 
Root cause: 
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:132)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeBeforeInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:84)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:606)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:674)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.2.1', revision: '16551', time: '2012-04-11 21:42:35'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0'
Driver info: driver.version: ScreenShotRemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:454)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:140)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:95)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:103)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.<init>(ScreenShotRemoteWebDriver.java:62)
	at com.paypal.test.bluefin.platform.grid.ScreenShotRemoteWebDriver.createInstance(ScreenShotRemoteWebDriver.java:400)
	at com.paypal.test.bluefin.platform.grid.Grid.startSession(Grid.java:71)
	at com.paypal.test.bluefin.platform.grid.SeleniumGridListener.beforeInvocation(SeleniumGridListener.java:89)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://qa-autocon26.sc4.paypal.com:4444 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:303)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:282)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:432)
	... 18 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:75)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:337)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:198)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:180)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:157)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
]]>
    </failure>
  </testcase> <!-- chipTxnTest -->
</testsuite> <!-- com.paypal.test.riskmint.pph.base.PPHBaseRegressionTest -->
